src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -149,13 +149,18 @@ public String key() {
     /**
      * The system time in millis at which the current instance of PlaybackDelegate was instantiated
```

Comment:
This comment is not correct anymore as we're updating initialTime to a different value when `updatePlayheadPosition` is called.